### ISSUE TITLE
About load checkpoint

### ISSUE THREAD

**Issue by ztyxd** [DetectionTeamUCAS/RetinaNet_Tensorflow_Rotation#14]
when i load the resnet50_v1d chekpoint, it shows that : 

NotFoundError (see above for traceback)Restoring from checkpoint failed. This is most likely due to a Variable name or other graph key that is missing from the checkpoint. Please ensure that you have not altered the graph expected based on the checkpoint. Original error:

Tensor name "resnet50_v1d/C1/conv0/BatchNorm/beta" not found in checkpoint files './data/pretrained_weights/resnet50_v1d.ckpt'

I download the resnet50_v1.ckpt from the link in readme, can u give me some suggestions aobut this ?
thanks!

**Comment by yangxue0827** [comment]
check your pretrained weights path. @ztyxd

**Comment by ztyxd** [comment]
@yangxue0827  i got it, it is because the net-name in the cfg file, thanks a lot
